eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -106,7 +106,47 @@ if ($LatestsSorted[0] -ne $Version) {
 
 if ($ReplaceLatestEntryTitle)
```

Comment:
Is this the correct place to strip the headers? I suspect we call this with that flag for more then our SetPackageVersion script that we end up calling in Prepare-Release.

Reply:
I think it is. If we are replacing the lates entry header then it is most likely intended for release.